src/java.base/share/classes/jdk/internal/classfile/impl/StackMapGenerator.java

```diff
@@ -438,6 +440,10 @@ private void processMethod() {
         }
     }
 
+    private static ClassFormatError classFormatError(int thisOffset) {
+        throw new ClassFormatError(String.format("Bad stack map offset %d", thisOffset));
```

Comment:
```suggestion
        return new ClassFormatError(String.format("Bad stack map offset %d", thisOffset));
```
Prefer to return instead of throw in such convenience methods.

Reply:
The same here - we should not throw CFE anymore.